api/rpc/account/account.proto

```diff
@@ -171,7 +173,7 @@ message AccountRequest {
   string name = 1;
 }
 
-message Account {
+message AccountInfo {
```

Comment:
:+1:  I like the idea of using a separate package for schemas, but was not sure how we wanted to organize this